src/components/input/input.svelte

```diff
@@ -60,6 +61,37 @@
    */
   export let mode: Mode | undefined = undefined
 
+  type InputEventDetail = {
+    innerEvent: Event & { target: HTMLInputElement }
```

Comment:
Basically, because it's the event from the HTML button element . In Svelte the events don't have a target, but conceptually the event is coming from the `LeoButton`, rather than the internal HTML button. Happy to be convinced we should call it something else though :)